### PR TITLE
docs: update readme to describe Steward in details

### PR DESCRIPTION
**Pull request by dpal**
Signed-off-by: Dmitri Pal <dmitri@profian.com>

A high-level overview of the value and benefits of the Steward component.

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -1,6 +1,31 @@
 // SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
 // SPDX-License-Identifier: AGPL-3.0-only
 
+//!
+//!# Overview
+//!
+//!Steward is a critical element of the Confidential Computing infrastructure. The promise of Confidential Computing is fully utilized when the workload runtime (Enarx WebAssembly) deployed into a Trusted Execution Environment (TEE) is assessed and verified for correctness before an actual workload is released into a TEE from the registry (Drawbridge). An external **attestation service** must perform evidence verification and assessment of the hardware's trustworthiness.
```
Comment (npmccallum) [inline src/main.rs:7]: 1. You need a space after `//!`.
2. Lines should be wrapped at 80 characters.

**Review by npmccallum — CHANGES_REQUESTED** [review]

File: src/main.rs [hunk 2]
```diff
@@ -1,6 +1,31 @@
...
+//!
+//!# Design Materials
+//!
+//![Attestation Concept](https://hackmd.io/@enarx/r1Yg2kb_s)
+//![Attestation Flow](https://hackmd.io/@enarx/SySK2_tHo)
+//![Full Provisioning Flow with Attestation](https://hackmd.io/@enarx/rJ55urrvo)
+//!
+//!# Licensing and Copyright
+//!
+//!Contributions to this project require copyright assignment to Profian.
```
Comment (npmccallum) [inline src/main.rs:27]: This should mention AGPL-3.0 licensing.

File: src/main.rs (hunk 1 above)
Comment (npmccallum) [reply src/main.rs:7]: Do this for all the new documentation.

File: src/main.rs (hunk 2 above)
Comment (dpal) [reply src/main.rs]: The title and licensing are added automatically by the readme generator. The rest I tried to fix. Let us see.

**Review by rjzak — APPROVED** [review]

File: src/main.rs (hunk 2 above)
Comment (dpal) [reply src/main.rs]: The change is made.

File: src/main.rs (hunk 1 above)
Comment (dpal) [reply src/main.rs]: The change is made.

**Review by npmccallum — APPROVED** [review]

**Pull request merged**